salt/modules/zypper.py

```diff
@@ -326,7 +326,8 @@ def upgrade_available(name):
 
         salt '*' pkg.upgrade_available <package name>
     '''
-    return latest_version(name)
+    # The "not not" tactic is intended here as it forces the return to be False.
+    return not not latest_version(name)  # pylint: disable=C0113
```

Comment:
`bool(name)` would be more idiomatic.  `not not` is only necessary if you're supporting Python versions older than 2.2.1.